tests/ui/search.js

```diff
@@ -59,38 +41,36 @@ casper.test.begin('Search baseline tests', {
                 test.assertUrlMatch(/\/search\?q=test&device_override=desktop$/);
                 test.assertField('compatibility_filtering', 'desktop');
                 helpers.assertAPICallWasMade('/api/v2/fireplace/search/', {
-                    // Note: device_override is present in the query string because
-                    // we inject everything except 'src' from the page query string.
-                    cache: '1', dev: 'desktop', device_override: 'desktop', lang: 'en-US', limit: '25', q: 'test', region: 'us', vary: '0'
+                    // device_override present in query string because we
+                    // inject everything except 'src' from page query string.
+                    cache: '1', dev: 'desktop', device_override: 'desktop',
+                    lang: 'en-US', limit: '25', q: 'test', region: 'us',
+                    vary: '0'
                 });
 
-                // There should be 26 elements in the listing: 25 items + "load more".
+                // Should be 26 elements in results: 25 items + "load more".
```

Comment:
You might be thinking of dialog, where *could* have the period inside the quotation marks, but that's a stylistic preference. But in this case, it isn't dialog, the quotation marks signify an exact string.